content/reference/hoon/irregular.md

```diff
@@ -366,6 +377,81 @@ See [%sand](/reference/hoon/rune/constants#warm) for other irregular definitions
   "~[1 2 3]"
   ```
 
+### `,` com
+
+`,` can serve in several capacities in Hoon programs:
+
+1. As sugar for the `^:` ketcol or `$;` bucmic runes, toggling structure and value mode.
+    (Toggling out of structure mode is uncommon.)
+
+    ```
+    > !,(*hoon ,[@t @t])
+    [ %ktcl
+      p=[%bccl p=[i=[%base p=[%atom p=~.t]] t=[i=[%base p=[%atom p=~.t]] t=~]]]
+    ]
+    
+    > !,(*hoon |=(a=,[@t @t] b))
+    [ %brts
+        p
+      [ %bcts
+        p=term=%a
+          q
+        [ %bcmc
+          p=[%cltr p=[i=[%base p=[%atom p=~.t]] t=[i=[%base p=[%atom p=~.t]] t=~]]]
+        ]
+      ]
+      q=[%cnts p=~[[%.y p=2] %a] q=~]
+    ]
+    
+    > !,(*hoon ,,[@t @t])
+    [ %ktcl
+        p
+      [ %bcmc
+        p=[%cltr p=[i=[%base p=[%atom p=~.t]] t=[i=[%base p=[%atom p=~.t]] t=~]]]
+      ]
+    ]
+    ```
+    
+    (`$;` bucmic, or manual value mode, lets you "borrow the sample" from a gate,
+    for instance.  It is not commonly used.)
```

Comment:
I'd say something like "allows the use of 'value mode' syntax to construct a mold". Concretely, it lets you build a mold out of `hoon` instead of `spec`.